httplog/request_test.go

```diff
@@ -61,7 +61,21 @@ func TestRequestLogger(t *testing.T) {
 
 			got := logOutput.String()
 
-			assert.Equal(t, tt.want, got, "logs")
+			if len(got) < 1 {
+				assert.True(t, len(tt.want) < 1, "got empty data, expected logs")
+
+				return
+			}
+
+			result := make(map[string]any)
+			err := json.Unmarshal([]byte(got), &result)
```

Comment:
this won't catch the case where zerolog will emit duplicate keys when they are redundantly set if that's worth testing.

Reply:
I'm not sure there is value is guarding that.  Each log processor will have different rules for how it handles duplicate keys.  And this code does not duplicate the fields.